metaserver/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -248,15 +248,15 @@ class ScalametaLanguageServer(
       td: TextDocumentIdentifier,
       position: Position
   ): Hover = {
-    val path = Uri.toPath(td.uri).get.toRelative(cwd)
-    symbol.hoverInformation(path, position.line, position.character) match {
+    val path = Uri.toPath(td.uri).get
+    compiler.typeAt(path, position.line, position.character) match {
       case None => Hover(Nil, None)
-      case Some((pos, denotation)) =>
+      case Some(tpeName) =>
```

Comment:
Is it 100% sure that `tpeName` is never empty? 😄

Reply:
I tried hovering on everything and it never showed empty boxes 💃 

Of course I may be wrong ^^

Reply:
I would still add a check here, if it's not an a priori known fact that `tpeName` cannot be empty.

Another question here: why did you remove the range parameter?

Reply:
I didn't find an easy way to retrieve that from the PC. I admit I didn't look

Reply:
As per the check, it'd be weird that `tree.tpe.widen` would return an empty string. I suppose if it ever comes up we can handle it pretty easily :)

Reply:
Works great! 💯 So cool!
And yes, I thought that without range symbols won't be highlighted correctly, but both VS Code and Atom handle it very well.